clients/cloud/confluent-cli/README.md

```diff
@@ -108,7 +108,7 @@ $ ccloud topic create test2
 * `--property schema.registry.basic.auth.user.info`: <SR API KEY>:<SR API SECRET> 
 
 ```bash
-$ confluent local produce test2 --cloud --value-format avro --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"count","type":"int"}]}' --property schema.registry.url=https://<SR ENDPOINT> --property basic.auth.credentials.source=USER_INFO --property schema.registry.basic.auth.user.info='<SR API KEY>:<SR API SECRET>'
+$ confluent local produce -- test2 --cloud --value-format avro --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"count","type":"int"}]}' --property schema.registry.url=https://<SR ENDPOINT> --property basic.auth.credentials.source=USER_INFO --property schema.registry.basic.auth.user.info='<SR API KEY>:<SR API SECRET>'
```

Comment:
Is .a double-dash needed _before_ `test2` here?

Reply:
Nope, great catch again